grapher/scan.py

```diff
@@ -117,7 +117,7 @@ def pkgToUnit(pkg: Dict) -> List[Unit]:
             deps.append(dep)
 
     unit = Unit(
```

Comment:
`pkgToUnit` should be renamed to `pkgToUnits`, since it can return more than 1 unit now.